server/datastore/mysql/vpp.go

```diff
@@ -433,7 +433,7 @@ func (ds *Datastore) SetTeamVPPApps(ctx context.Context, teamID *uint, incomingA
 		// upsert it if it does not exist or labels or SelfService or InstallDuringSetup flags are changed
 		existingApp, isExistingApp := existingApps[incomingApp.VPPAppID]
 		incomingApp.AppTeamID = existingApp.AppTeamID
-		var labelsChanged, categoriesChanged, displayNameChanged bool
+		var labelsChanged, categoriesChanged, displayNameChanged, configurationChanged bool
```

Comment:
we don't have to do it here, but we should probably extract all this "did it change" logic into a function or method.